src/libbson/doc/bson_array_as_json.rst

```diff
@@ -22,7 +22,7 @@ Description
 
 The :symbol:`bson_array_as_json()` function shall encode ``bson`` as a UTF-8
 string using libbson's legacy JSON format, except the outermost element is
-encoded as a JSON array, rather than a JSON document.
+encoded as a JSON array, rather than a JSON document. This function is superseded by :symbol:`bson_array_as_canonical_extended_json()` and :symbol:`bson_array_as_relaxed_extended_json()`, which use the same `MongoDB Extended JSON format`_ as all other MongoDB drivers.
```

Comment:
```suggestion
encoded as a JSON array, rather than a JSON document. This function is
superseded by :symbol:`bson_array_as_canonical_extended_json()` and
:symbol:`bson_array_as_relaxed_extended_json()`, which use the same
`MongoDB Extended JSON format`_ as all other MongoDB drivers.
```

Missing the following line at the bottom of this file, without which the `MongoDB Extended JSON format` reference is invalid:

```rst
.. _MongoDB Extended JSON format: https://github.com/mongodb/specifications/blob/master/source/extended-json.rst
```